examples/flask/app_v3.py

```diff
@@ -0,0 +1,99 @@
+# Note that this file contains some inline comments starting with # --, used to
+# generate documentation from this file. You're probably better served reading
+# the actual documentation (see Using in Flask in the docs).
+
+from flask import Flask
+from flask_restful import Api
+from doctor.errors import NotFoundError
+from doctor.router_v3 import create_routes, get, post, put, delete
+from doctor.types import array, boolean, integer, string, Object
+
+# -- mark-types
+
```

Comment:
@noonat had mentioned this looks a little off as it looks like builtin types at first glance.  A suggestion for overcoming that is to just import types and change this to look like:

```python
from doctor import types

Body = types.string('Note body')
# ... etc.
```